File: test/fixtures/elixir/post_escaped_single_quotes_in_double_quotes.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://example.com/",
  options: [],
  headers: [
    {~s|Content-Type|, ~s|application/x-www-form-urlencoded|},
  ],
  params: [],
  body: [
    {~s|foo|, ~s|\\\'bar\\\'|}
  ]
}

response = HTTPoison.request(request)

File: test/fixtures/php-guzzle/post_escaped_single_quotes_in_double_quotes.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->post('http://localhost:28139/', [
    'headers' => [
        'Content-Type' => 'application/x-www-form-urlencoded'
    ],
    'body' => "foo=\\'bar\\'"
]);
